src/lib/AppRegistry.tsx

```diff
@@ -12,19 +10,18 @@ import {
   InboxRenderer,
   InquiryRenderer,
   MyProfileRenderer,
-  RenderCallback,
-  SaleRenderer,
   WorksForYouRenderer,
 } from "./relay/QueryRenderers"
 import FavoritesScene from "./Scenes/Favorites"
 import HomeScene from "./Scenes/Home"
 import renderWithLoadProgress from "./utils/renderWithLoadProgress"
-import { Schema, screenTrack as track, Track } from "./utils/track"
+import { Schema, screenTrack as track } from "./utils/track"
 
+// FIXME: This isnt being used
```

Comment:
In some of the more ambiguous "WIP?" areas I've commented out and added a note

Reply:
This can go, the root component decorator is called screenTrack.